README.md

```diff
@@ -1,21 +1,51 @@
 # DUST
 
-This repo hosts the contracts for the DUST world.
+Dust is our serious attempt at seeding an autonomous world.
 
-### Running
+## DUST is live on Redstone mainnet, but still in alpha.
 
-```
-pnpm run dev
-```
+What you create in Dust is real and permanent. The map and your progress won’t be reset.
+
+However, Dust is still in alpha and focused on balancing and stability. Some parts of the system may evolve, but always with full transparency and a clear path toward decentralization.
+
+🔒 **Things that will not change anymore:**
+
+- The current map, both natural terrain and structures created by players.
+- The number of items in your inventory and chests.
+- The world contract address. If systems within the world are updated, their interfaces will be backwards compatible, so your programs don’t break.
+
+🚧 **Things that may change during alpha:**
 
-### Installation
+- Balancing changes, i.e. cost or requirements of actions, block mass, drop probabilities.
+- Introduction of new game systems or items.
+- Bug fixes and stability improvements.
+- Map extensions beyond the current borders.
 
-Make sure sure you have:
+The goal of all changes is to put Dust on the path of autonomy. Changes will be communicated in advance and can be followed in this repository. The changelog includes a historical log of changes since the mainnet deployment.
```

Comment:
```suggestion
The goal of all changes is to put DUST on the path of autonomy. Changes will be communicated in advance and can be followed in this repository. The changelog includes a historical log of changes since the mainnet deployment.
```